feat(Unity): Added WebGL limitations (#13637)
getsentry/sentry-docs: docs/platforms/unity/troubleshooting/known-limitations.mdx

## Cysharp

Currently, the SDK fails to provide line numbers and source context in combination with Cysharp. For context, the SDK relies on Unity's IL2CPP backend to provide it with instruction addresses for Sentry generate line numbers in events. When using Cysharp, those instruction addresses default back to `0x0` once the stacktrace goes into the library. There is no known workaround for this.

## WebGL Support

> When targeting WebGL, users have the option of setting the `PlayerSettings.WebGL.exceptionSupport`. There are currently the following limitations to be aware of:

Reviewer: ```suggestion
When targeting WebGL, you have the option of setting the `PlayerSettings.WebGL.exceptionSupport`, but be mindful of the following limitations:
```
